packages/cozy-sharing/src/components/Recipient/OwnerRecipient.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { useClient, useQuery, hasQueryBeenLoaded } from 'cozy-client'
+
+import Recipient from './Recipient'
+import { buildInstanceSettingsQuery } from '../../queries/queries'
+
+const DefaultOwnerRecipient = () => {
+  const client = useClient()
+
+  const instanceSettingsQuery = buildInstanceSettingsQuery()
+  const instanceSettingsResult = useQuery(
+    instanceSettingsQuery.definition,
+    instanceSettingsQuery.options
+  )
+
+  const public_name = instanceSettingsResult?.data?.attributes?.public_name
```

Comment:
nit: use Camelcase for naming this variable even if propertie from attributes is in Snakecase to consistency